samples/openthread/ncp/overlay-logging.conf

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+# option for configuring log level in sample module
```

Comment:
Please consider starting the comment lines with upper case. Also remove the # for the empty line above.